kaggle_environments/envs/chess/chess.py

```diff
@@ -115,12 +115,14 @@ def is_insufficient_material(board):
 
     for square in range(64):
         piece = board.get_piece(square)
-        if piece:
+        if piece and piece != " ":
             if piece.isupper():
                 white_pieces[piece.lower()] += 1
             else:
                 black_pieces[piece.lower()] += 1
 
+    print(white_pieces, sufficient_material(white_pieces))
+    print(black_pieces, sufficient_material(black_pieces))
```

Comment:
nit: typically best to remove `print` statements if only used for debugging while developing